LIB/SCREEN22.GEN

```diff
@@ -10,7 +10,7 @@
 ; Usa fun.: BIOS call interslot
 
 SCREEN	EQU 05Fh
-CALSLT	EQU 001Ch
+;CALSLT	EQU 001Ch
```

Comment:
https://github.com/Fubukimaru/asMSX/blob/a3663eb8fdc9581e51fca03baf1101f1dc674d86/src/dura.y#L3407

`CALSLT` is already defined in asMSX code.